Lab2/scripts/deployment.sh

```diff
@@ -19,6 +19,17 @@ while [[ "$#" -gt 0 ]]; do
     shift
 done
 
+# Check if workshop is running through EventEngine or not
```

Comment:
might be good to add details that during an AWS hosted event we pre-provision Cloudfront and S3 to save time and hence this code

Reply:
Done.